### ISSUE TITLE
New Term - sensitiveVernacularName

### ISSUE THREAD

**Issue by qgroom** [tdwg/dwc#522]
## New term

* Submitter: Quentin Groom, Ina Vandebroek

* Efficacy Justification (why is this term necessary?): In light of the XX International Botanical Congress held in 2024 in Madrid, Spain, and ongoing discussions within the biodiversity community, there is increased awareness of the use of insulting, derogatory, and inappropriate vernacular names. These names can often be used unintentionally, and it is important for users of names to be properly informed about potentially offensive terms.

* Demand Justification (name at least two organisations that independently need this term):
New York Botanical Garden
U.S. Botanic Garden
Meise Botanic Garden
University of the West Indies, Mona

* Stability Justification (what concerns are there that this might affect existing implementations?): none
* Implications for dwciri: namespace (does this change affect a dwciri term version)?: No

Proposed attributes of the new term: text

* Term name (in lowerCamelCase for properties, UpperCamelCase for classes): sensitiveVernacularName

* Term label (English, not normative): Sensitive vernacular name information

* Organized in Class (e.g., Occurrence, Event, Location, Taxon): Taxon

* Definition of the term (normative): An advisory that the vernacular name mentioned in dwc:vernacularName may be considered inappropriate for several reasons, including but not limited to words or their derivations that evoke: cultural sensitivity, ethic or racial sensitivity, gender sensitivity, derogatory language, religious sensitivity, colonial or imperial legacy, inaccuracy or misrepresentation, offensive language, cultural appropriation.

* Usage comments (recommendations regarding content, etc., not normative): Details on whether a vernacular name might be considered offensive in certain contexts. Specify the relevant contexts if applicable. If research does not indicate any sensitivity related to the name, the best practice is to enter 'unknown' in this field.
* Examples (not normative): “This name is consider a racial slur in English speaking countries”

* Refines (identifier of the broader term this term refines; normative): 

* Replaces (identifier of the existing term that would be deprecated and replaced by this term; normative): N/A

* ABCD 2.06 (XPATH of the equivalent term in ABCD or EFG; not normative): none

**Comment by Jaim3Fry3** [comment]
This is an excellent, and important way to increase transparency and communication about names with the potential to cause harm that does not restrict usage but allows institutions to make their own, informed decisions. I wholeheartedly support this term addition!

**Comment by nielsklazenga** [comment]
This does not make a whole lot of sense. Darwin Core does not have a Name object and the best way to deal with inappropriate labels on a Darwin Core Taxon is to just not use them. One could use the `informationWithheld` property to indicate that the vernacular name that was used was withheld because it was deemed inappropriate.

**Comment by qgroom** [comment]
> This does not make a whole lot of sense. Darwin Core does not have a Name object and the best way to deal with inappropriate labels on a Darwin Core Taxon is to just not use them. 

It seems that you may not fully grasp the intended use case. Many vernacular names carry offensive or derogatory connotations, often specific to a particular group, language, or cultural context. To address this, several organizations have compiled catalogues or checklists of such names to inform others, ensuring that people do not unknowingly use terms with negative or harmful meanings.

This term is particularly relevant for people working in museums, botanical gardens, and other institutions where multilingual labels and interpretive materials are created. Similarly, it can benefit those writing floras and faunas by helping them understand the broader context of the names they include in their work.

In Darwin Core, taxonomic names are represented within the "Taxon" class, and the Darwin Core schema facilitates the exchange of name-related information.

This proposal originated from a symposium at the XX International Botanical Congress (Towards a more ethical science: Decolonizing Botanical Research) and during the Nomenclatural session of the Congress, where the issue of derogatory Latin names was a key topic of discussion. I have not addressed issues related to Latin names here, as that is a rather more complicated subject, with potentially different solutions.

**Comment by nielsklazenga** [comment]
Sorry, makes even less sense now. I understand the context, but I do not see a use case for this term in it. Inappropriate labels on specimens might be sensitive data, but they are not sensitive vernacular names. `vernacularName` is interpreted data. You decide what is a vernacular name and what is not, so if you think that a label is inappropriate, you just do not record it as a `vernacularName`, but you record it in `fieldNotes` or `identificationRemarks` if you want to exchange it at all.

It also seems counter-productive to broadcast sensitive data to the world by having special properties for it. I think for this kind of sensitive data the more appropriate options are to either withhold it or annotate it.

**Comment by qgroom** [comment]
Some additional resources...

- https://www.publicgardens.org/wp-content/uploads/2023/10/potentially-problematic-common-names-north-american-public-gardens.pdf
- https://ibcmadrid2024.com/?seccion=scientificArea&subSeccion=detailAbstract&id=3505
- https://ibcmadrid2024.com/?seccion=scientificArea&subSeccion=detailAbstract&id=2455

**Comment by nielsklazenga** [comment]
@qgroom, just to clarify my major issue with this proposal, a scientific name or vernacular name on a specimen label is not a `dwc:scientificName` or `dwc:vernacularName`, but a `dwc:Identification`. I am also not in favour of a `dwc:sensitiveVerbatimIdentification`, as I think there are better solutions for your use case outside Darwin Core, but it would at least say what (I think) you mean.

**Comment by nielsklazenga** [comment]
@Jaim3Fry3 , I never said that. I said 'withhold or annotate' and 'annotating' is what you call 'Highlighting these concerns in a clear, consistent manner'. And until you can do that and data consumers can deal with it, you withhold (or maybe I should say 'hold back') the data you consider might be sentitive or you do not yet know what to do with. At institutions like the one I work at people make these sort of decisions all the time.

On the other hand, 'Highlighting these concerns in a clear, consistent manner' is exactly what this proposal does not do, as it does address only one very specific piece of potentially sensitive information. The proposal does not address the stated use case, as it addresses the wrong term (or it least one of us thinks so) and also, because it focuses on one specific term rather than all the entire label, the `sensitiveVernacularName` disappears into the Identification History as soon as there is a new Identification while the sensitive bit of data is still on the label and on the image of the specimen.

The thing with sensitive data is that some consumers want to have it and other consumers do not (or should not). Also, different producers will consider different things sensitive. Therefore, sensitive data should be dealt with at the record or data set level, not the term level. The `informationWithheld` might not be sufficient, but it is in record-level terms like that that the solution should be sought. This is what the [Sensitive Species Extension Task Group](https://www.tdwg.org/community/dwc/sensitive-species/) is looking at for the type of sensitive data that is generally dealt with with `dataGeneralizations` (and since we apparently want to be consistent...).

**Comment by tucotuco** [comment]
There are many Darwin Core terms that contain sensitive content in the sense being used for this term proposal. I support @nielsklazenga approach over also needing to add sensitiveLocality, sensitiveVerbatimLocality, sensitiveRecordedBy, sensitiveCaste, sensitiveBehavior, sensitiveOccurrenceRemarks, sensitiveOrganismName, sensitiveOrganismRemarks...

**Comment by nielsklazenga** [comment]
If @qgroom agrees, maybe this proposal can be converted to a discussion where we can discuss how to deal with this type of sensitive data (as opposed to sensitive data that can be generalised)?

**Comment by ArthurChapman** [comment]
I agree with @tucotuco. I understand that a Task Group is being established at TDWG to look at, and recommend Darwin Core Terms for sensitive data.

**Comment by qgroom** [comment]
> There are many Darwin Core terms that contain sensitive content in the sense being used for this term proposal. I support @nielsklazenga approach over also needing to add sensitiveLocality, sensitiveVerbatimLocality, sensitiveRecordedBy, sensitiveCaste, sensitiveBehavior, sensitiveOccurrenceRemarks, sensitiveOrganismName, sensitiveOrganismRemarks...

As many of you know last year, @ArthurChapman wrote an in-depth report on sensitive primary species occurrence data, including recommendations for changes to Darwin Core. It’s important to note that he did not address the issue of derogatory vernacular names, as this is a separate topic. Similarly, the Sensitive Species Extension Task Group does not have this issue within its remit.

The proposed `sensitiveVernacularName` term caters to different use cases and has a distinct stakeholder groups.

The word "sensitive" seems to have led to confusion between these unrelated issues. If you can suggest an alternative word it might be better. e.g. flaggedVernacularName, sensitiveLanguageVernacularName

Regarding the formation of a task group, it’s generally more effective to explore other solutions first. Task groups often struggle with efficiency and participation and can sometimes hinder standardization efforts. Unless there is a significant and controversial workload to address, it might be best to avoid forming one.

Chapman AD (2020) Current Best Practices for Generalizing Sensitive Species Occurrence Data.
Copenhagen: GBIF Secretariat. https://doi.org/10.15468/doc-5jp4-5g10.
```

**Comment by nielsklazenga** [comment]
@qgroom , I agree that this is out of scope for the [Sensitive Species Extension Task Group](https://www.tdwg.org/community/dwc/sensitive-species/) (which is already there), but having special terms for different use cases and stakeholders kind of defeats the purpose of having a standard.

**Comment by tucotuco** [comment]
Why just vernacular names? Sensitive language permeates biodiversity data. Why not have a solution that can be applied to any or all of it, in a record-level term?

**Comment by qgroom** [comment]
> Why just vernacular names? Sensitive language permeates biodiversity data. Why not have a solution that can be applied to any or all of it, in a record-level term?

The primary reason for focusing on vernacular names is that Darwin Core is specifically designed for exchanging taxon-based checklist data, which includes both scientific and vernacular names. Unlike place names, which can also have cultural sensitivities, we do not typically use Darwin Core for gazetteer information exchange.

Latin names present their own challenges, but we cannot arbitarily choose which Latin name to use, Though one could argue it would be good to exchange information on this too. However, Latin names are controlled by the Nomenclatural Codes, so we should really take a lead from the Codes.

There are other fields that may contain offensive remarks, but that strays into a different use case. One case that Niels has mentioned is occurrence data. The intension here is to have the ability to communicate about the sensitivity of certain names and under what context. Not to judge that it is offensive or derogatory, as that is often a matter of perspective, but to ensure this information can be shared.

**Comment by nielsklazenga** [comment]
Sorry, I misunderstood the use case. I still do not understand why you cannot use a general-purpose record-level term, as they would be available in the Taxon Core and the Vernacular Name Extension as well.

**Comment by nielsklazenga** [comment]
> The intension here is to have the ability to communicate about the sensitivity of certain names and under what context.

It will not be in the initial release of TCS 2, because it is not in TCS 1, but I have plans for a VernacularName class in TCS, which would be pretty much the same as the [GBIF VernacularName](http://rs.gbif.org/terms/1.0/VernacularName). So, if this can wait and people can use `taxonRemarks` in the [Vernacular Name Extension](https://rs.gbif.org/extension/gbif/1.0/vernacularname.xml) for now, I think that might be a good context for this discussion.

**Comment by MattBlissett** [comment]
> The word "sensitive" seems to have led to confusion between these unrelated issues. If you can suggest an alternative word it might be better.

I think "taboo" fits what's being discussed, "tabooVernacularName", and avoids the clash with the usual biodiversity meaning of a sensitive species.

**Comment by gkampmeier** [comment]
I agree that sensitive can be confusing.  I think that "deprecated" seems more fitting. It firmly puts the context as something that is no longer used and that the reason may be that it is "taboo" but there could be other shades of reasons why it may exist, but should no longer be used (plus people often like to break taboos ;).

**Comment by TaniaGLaity** [comment]
wrt the https://www.tdwg.org/community/dwc/sensitive-species/ we are also interested in tackling the problem of data which may be sensitive for cultural reasons.  we are not only considering obfuscation / generalisation of occurrence records but also withholding some record level / dataset level attributes where they are considered sensitive.

**Comment by ben-norton** [comment]
A couple questions:
1. Who determines what is and what is not culturally sensitive? 
2. What happens if there is a dispute?
3. What is the process for determining whether something is offensive?
My larger point is that, as a concept, this makes sense and is commendable. In practice, however, it may become a highly subjective point of contention that accomplishes the exact opposite of the intent.

**Comment by ben-norton** [comment]
Good question Quentin. I know the term has only good intentions. I share the sentiment. Here's the problem. From the proposal, the intent of the term is to highlight terms that someone might find offensive, which is effectively an awareness campaign. First, it's certainly a worthy cause. Second, I don't think Darwin Core is the right medium for such an exercise. 
Here are a couple scenarios.
1. I download a dataset. In the dataset, there is a note that a vernacular name is considered offensive by a given population. Now, I have to make a decision whether or not to use the data, which is not based on science or data quality assessment but rather an exercise in culture sensitivity. Now that it's been highlighted, am I guilty of distributing offensive material if I choose to use it? That's not a question we should ask when someone downloads a Darwin Core dataset. It's important, but better addressed in a different medium than a Darwin Core dataset. I don't want to offend anyone. I'm just looking to use a dataset for a research exercise. Now I have to ponder whether doing so is somehow going to offend others. It may be easier to just stop using Darwin Core datasets. That's the only sure way to know I won't offend anyone. It sounds a little ridiculous, but holds true. 
2. Let's say I publish a dataset. In that dataset, there is a vernacular name that some populations may find culturally sensitive. I am completely unaware of this. I then inadvertently leave the sensitive field empty.  By not declaring the sensitivity, am I accountable for the negative effects the dataset may have on certain populations? What is my liability here? 
Do we really want to be going down this road with Darwin Core?
The overarching concern is that when you address an issue in the wrong medium, you are not addressing it in the right medium. So despite the best of intentions, the end result is actually counterproductive. 
I think an awareness campaign regarding the usage of sensitive vernacular names should be undertaken. It's a great idea, just not in a Darwin Core dataset.

**Comment by ManonGros** [comment]
Wouldn't an additional to the vernacularName extension be more suited? This would allow to define the context (language, country, region, etc.) of the sensitive name.

**Comment by ben-norton** [comment]
@ManonGros Awesome.

**Comment by themerekat** [comment]
> Why just vernacular names? Sensitive language permeates biodiversity data. Why not have a solution that can be applied to any or all of it, in a record-level term?

I agree with @tucotuco here.

Brilliant suggestion @ManonGros that this could be part of the vernacularName extension.

**Comment by tucotuco** [comment]
> > Why just vernacular names? Sensitive language permeates biodiversity data. Why not have a solution that can be applied to any or all of it, in a record-level term?
> 
> I agree with [@tucotuco](https://github.com/tucotuco) here.

Such a term could be modeled on informationWithheld, and dataGeneralizations. I would call it dataSensitivity, or something like that.

> > Wouldn't an additional term to the vernacularName extension be more suited? This would allow to define the context (language, country, region, etc.) of the sensitive name.
>
> Brilliant suggestion [@ManonGros](https://github.com/ManonGros) that this could be part of the vernacularName extension.

The vernacularName extension is not part of any standard. Adding the term there would have no effect on Darwin Core, nor would it require a Darwin Core public review. That doesn't mean that both things couldn't happen (add the term to the standard and add the standard term to the extension).

**Comment by TaniaGLaity** [comment]
we are discussing these and similar use cases in the Sensitive Data Extension Task group - https://www.tdwg.org/community/dwc/sensitive-species/ 
we are meeting this week to go through our draft list of sensitivity reasons and treatments and use cases.
We have yet to work out the implementation of this but could be something similar to how measurement or fact is implemented?
but in saying that there are a lot of great alternatives to that idea in this thread!

**Comment by qgroom** [comment]
It might be useful if people read back through the thread, because many of this issues have been mentioned before.

It has been agreed that this is out of scope for the sensitive species task group.

As I mentioned earlier...
> The word "sensitive" seems to have led to confusion between these unrelated issues. If you can suggest an alternative word it might be better. e.g. flaggedVernacularName, sensitiveLanguageVernacularName

Latin names are out of our hands too, as they are controlled by the International Codes of nomenclature and we can't avoid them whether they are offencive or not.

**Comment by qgroom** [comment]
> I download a dataset. In the dataset, there is a note that a vernacular name is considered offensive by a given population. Now, I have to make a decision whether or not to use the data, which is not based on science or data quality assessment but rather an exercise in culture sensitivity. Now that it's been highlighted, am I guilty of distributing offensive material if I choose to use it? That's not a question we should ask when someone downloads a Darwin Core dataset. It's important, but better addressed in a different medium than a Darwin Core dataset. I don't want to offend anyone. I'm just looking to use a dataset for a research exercise. Now I have to ponder whether doing so is somehow going to offend others. It may be easier to just stop using Darwin Core datasets. That's the only sure way to know I won't offend anyone. It sounds a little ridiculous, but holds true.

You seem to be suggesting it is better to remain ignorant to avoid having to make any decisions. Also, no one is suggesting ditching whole datasets, just informing people of the contexts where names might be offencive. If you learn a vernacular might be offensive then don't use it.

> Let's say I publish a dataset. In that dataset, there is a vernacular name that some populations may find culturally sensitive. I am completely unaware of this. I then inadvertently leave the sensitive field empty.  By not declaring the sensitivity, am I accountable for the negative effects the dataset may have on certain populations? What is my liability here?

Not any more liable than anyone else who doesn't know something. People publishing vernacular names should do their due diligence, but they also need sources to work with. Dictionaries do not shy away from including offencive words. Context is important. Informing people of potentially offensive names is not going to have negative effects on a population. Allowing others to be ignorant of those names would have more impact.

>   Do we really want to be going down this road with Darwin Core?

Darwin Core is specifically used to communicate data on names of organisms. I can't think of a better place.

**Comment by TaniaGLaity** [comment]
We have included offensive terms in our sensitivity reasons so that we can include it in our discussions in tomorrow's meeting.  I think it's worth including as it has been mentioned before wrt place names, vernaular names or other descriptive text.  The idea would be to exclude it, as you said with a note that it has been excluded or redacted (similar to how sensitive locality or habitat attributes are to replaced) whilst making it clear that this has been replaced / excluded / redacted based on XX reason according to XX person / institution / community / group.  That way a data requestor knows that there is additional information potentially available on request and under certain conditions

**Comment by ArthurChapman** [comment]
Noyt sure that "sensitive" is the correct term here. What is really being discussed is "offensive" which can be quite subjective. I agree iwth Quentin that it doesn't fit at all with the Sensitive Data Extension Task group

**Comment by TaniaGLaity** [comment]
Ok I'll remove it.  Changed my mind - thanks @qgroom and @ArthurChapman et al - it's not about sensitive species data - it's more about other information associated with a record that will cause offence

**Comment by MattBlissett** [comment]
I think ongoing confusion with the word "sensitive" is likely, and it would be better if the term used a clearer name, like "offensiveVernacularName" or "inappropriateVernacularName".

**Comment by lucyschrader** [comment]
Yeah, seems like the sensitive species connotations for 'sensitive' would keep raising misunderstandings about the field.

Structurally I see this idea as something that contextualises another field, a little bit like how you need `organismQuantityType` to contextualise `organismQuantity`. You could accompany any applied name string field (`vernacularName`, `scientificName`, `specificEpithet`) with `xNameContext` (if you want it to be as neutral as possible), eg:

```
scientificName: Myosotis inappropriatename
scientificNameContext: this one's really not good

vernacularName: Dragonfly
vernacularNameContext: dragons are sadly not real
```

What to actually call it I'm not sure. xNameUsageNotes, xNameHistoryContext, xNameCulturalContext... If there's a way to do this that doesn't involve creating a whole suite of new terms that would also be good. When using a nested data structure that's easy but I suppose dwc expects to be flat.

**Comment by qgroom** [comment]
I like nameUsageNotes or nameCulturalContext!

**Comment by ManonGros** [comment]
I still don't understand why it should be a Taxon Core term. How would you handle the context associated with different vernacular names for a taxon?

For example:
`vernacularName: Dragonfly in English | Libellules for the adult form in French | naïade for the Larvae in French`
`vernacularNameContext: dragons are sadly not real | from the roman libra, a Roman unit of measure, equal to twelve ounces; a pound (abbreviated lb.). | river nymph in the greco-roman mythologies`

**Comment by tucotuco** [comment]
There seems to be general support for creating the capacity to indicate problematic vernacularNames, but there has been no consensus on how to accomplish that. I have added the new label "Unresolved" so as not to misrepresent the issue as "Controversial". This proposal will remain open for public review, but will not be included in the current move to the Executive Review phase.

**Comment by tucotuco** [comment]
This issue has had no additional activity during the DwC-DP public review. I have maintained the label "Unresolved" and have moved the issue to the next public review.
